itou/www/job_seekers_views/views.py

```diff
@@ -225,27 +225,47 @@ def get(self, request, *args, **kwargs):
         return HttpResponseRedirect(reverse(view_name, kwargs={"session_uuid": self.job_seeker_session.name}))
 
 
-class JobSeekerBaseView(TemplateView):
+class ExpectedJobSeekerKindMixin:
```

Comment:
```suggestion
class ExpectedJobSeekerSessionMixin:
```
? C'est d'ailleurs le nom que tu as mis dans la description de la PR et il me semble plus adapté.